packages/components/src/components/Textarea/index.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useState, useCallback } from 'react';
+import styled from 'styled-components';
+import css from '@styled-system/css';
+import { VisuallyHidden } from 'reakit/VisuallyHidden';
+import { uniqueId } from 'lodash-es';
+import { Text } from '../Text';
+import { Stack } from '../Stack';
+
+const placeholderStyles = {
+  color: 'input.placeholderForeground',
+  fontSize: 13,
+};
+
+export const TextareaComponent = styled.textarea(
```

Comment:
Should we reuse styles from Input with `styled(Input)`

Reply:
Done!